### PR TITLE
Better init script + better logger = better main.py

### PR DESCRIPTION
**Pull request by 45-razrblds**
Just adds a better logger, colorcoding and an init script that prints some sysinfo.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -3,37 +3,93 @@
...
+        tasks = []
         for file in os.listdir(self.cogsfolder):
-            if file.endswith(".py"): 
-                await self.load_extension(f"{self.cogsfolder}.{file[:-3]}")
+            if file.endswith(".py"):
+                cog_name = f"{self.cogsfolder}.{file[:-3]}"
+                tasks.append(self._load_cog(cog_name))
+
+        # Run tasks concurrently
+        await asyncio.gather(*tasks)
```
Comment (namooTH) [inline main.py:83]: this looks pretty unsafe in my opinion and cogs usually takes around less than a second to load

**Review by namooTH — CHANGES_REQUESTED** [review]
also wavelink setup seems to be missing, it is needed for music commands and its localhost so it doesnt matter if the password gets leaked

File: main.py (hunk 1 above)
Comment (45-razrblds) [reply main.py:83]: yeah, i kinda wanted to make some "ultrafast" thing, you can remove that if needed

**Comment by namooTH** [comment]
the `wavelink` setup function is kinda horrible right now, so if there's a better way to put it let me know

**Pull request merged**